core/domain/services/graphql/connections/TicketPricesConnection.php

```diff
@@ -76,39 +76,47 @@ public static function get_connection_args($args = [])
     {
         return array_merge(
             [
-                'ticket' => [
+                'in'              => [
+                    'type'        => ['list_of' => 'ID'],
+                    'description' => esc_html__('Limit pices to the given globally unique IDs', 'event_espresso'),
```

Comment:
ahhhh... was sooooo close to being finished without finding a single thing to comment about ☹️ 

```suggestion
                    'description' => esc_html__('Limit prices to the given globally unique IDs', 'event_espresso'),
```